docs/licensing.md

```diff
@@ -29,11 +29,29 @@ This document provides an overview of the licenses of important products created
     </dd>
 </dl>
 
-:::info
-If you are interested in using our products for your own solutions and code analysis tools, see the [Semgrep FAQ page](/faq/#how-are-semgrep-and-its-rules-licensed) for more information.
-:::
+## Claim a license
+
+Once you've purchased a subscription, you should receive an email from Semgrep
+with your license information. Follow the instructions provided in the email to
+claim your license and begin onboarding your Semgrep products.
+
+If you have been provided a license key by Semgrep or if you would like to claim a license manually:
+
+1. Sign up or log in to your Semgrep account.
+2. Create an org if you haven't already done so.
+3. Navigate to `http://semgrep.dev/orgs/-/settings/upgrade/YOUR_LICENSE_KEY`,
+   making sure that you REPLACE THE `YOUR_LICENSE_KEY` placeholder with your
```

Comment:
```suggestion
   making sure that you **replace** the `YOUR_LICENSE_KEY` placeholder with your
```
don't feel strongly about this, but this seems a bit better than using caps, so that the placeholder stands out more